bundles/models/models_service.go

```diff
@@ -162,6 +199,19 @@ func (ms *Service) ModelList(p *gz.PaginationRequest, tx *gorm.DB, owner *string
 		modelsProto.Models = append(modelsProto.Models, fuelModel)
 	}
 
+	// Cache the result if it's a basic query.
+	if basicQuery {
+		paginationBytes, _ := json.Marshal(paginationResult)
+		modelsBytes, _ := proto.Marshal(&modelsProto)
```

Comment:
Let's please handle the errors.

Reply:
Handled in [ca689c0](https://github.com/gazebo-web/fuel-server/pull/21/commits/ca689c01570d686f24b99050b213273719ec92fc)